db/models/index.js

```diff
@@ -11,43 +11,57 @@ const NUM_GAMES_PER_TRAIT = 4;
  */
 const fetchTraitsForProduct = async (id) => {
   const edgesCollection = db.collection('edges');
-  id = 'games/' + String(id);
+  id = 'games/' + String(id); // the id in arango is 'collection/id_string'
 
-  let results = await db.query(aql`
+  try {
+    let results = await db.query(aql`
         FOR doc in any ${id} ${edgesCollection}
           OPTIONS { bfs: true, uniqueVertices: 'global' }
           return { id: doc._id, name: doc.name }
     `);
 
-  results = await results.all();
-  console.log(results);
+    results = await results.all();
 
-  return results;
+    return results;
+  } catch (err) {
+    console.log('fetchTraitsForProduct: Error getting products from database.');
```

Comment:
Might want to include the err in your console.log!

Reply:
LOL. Good point. Thanks.